### ISSUE TITLE
GCC 4.9.3: Unable to find a usable ISL.

### ISSUE THREAD

**Issue by szechyjs** [crosstool-ng/crosstool-ng#173]
I'm building a toolchain with gcc v4.9.3, it seems the default ISL is 0.15 which does not work.

```
[CFG  ]    checking for version 0.10 of ISL... no
[CFG  ]    checking for version 0.11 of ISL... no
[CFG  ]    checking for version 0.12 of ISL... no
[CFG  ]    checking for version 0.14 of ISL... no
[ERROR]    configure: error: Unable to find a usable ISL.  See config.log for details.
```

Manually setting ISL to 0.14 fixes the problem.

**Comment by bhundven** [comment]
Thanks for letting us know! I knew this would happen. 😀

**Comment by jasmin-j** [comment]
Uups, I had the same issue yesterday, but I didn't think it is a big
problem, because the message says what you have to do.

> Thanks for letting us know! I knew this would happen. 😀
I can fix it, but not before Sunday when I am on vacation.

**Comment by jasmin-j** [comment]
@szechyjs:
Can you check if commit ed647a79b0cc567e620bf32a82098705ba598087 fixes your problem.

**Comment by jasmin-j** [comment]
Fixed by merge #175.
